src/interactableComponents/drawer/newDrawer.js

```diff
@@ -25,7 +25,8 @@ const ITEM_PROP_TYPES = {
 
 /**
  * @description: Drawer Component
```

Comment:
I already fixed this in master, please remove it from the PR

Reply:
Merged, done